gordo/cli/client.py

```diff
@@ -185,20 +203,27 @@ def metadata(ctx: click.Context, output_file: typing.Optional[typing.IO[str]]):
 
 @click.command("download-model")
 @click.argument("output-dir", type=click.Path(exists=True))
+@click.option(
+    "--target",
+    help="A list of machines to target. If not provided then target all machines in the"
+    " project",
+    default=[],
+    multiple=True,
+)
```

Comment:
Rather than repeating these blocks, do you think it's worth it to pass it higher up in the context? Maybe the ergonomics it worth it though?

Reply:
:man_shrugging:  Yeah maybe, but if felt a bit ugly having to dig it out, and with this there is a more clear mapping between the CLI and the python API